src/client/App/index.jsx

```diff
@@ -1,26 +1,54 @@
 import React from 'react';
-import { connect } from 'react-redux';
-import { renderRoutes } from 'react-router-config';
+import GSAP from 'react-gsap-enhancer';
+import { TweenMax, TimelineMax } from 'gsap';
 
-import './assets/index.scss';
+function createAnim() {
+  const box = document.getElementById('box');
+
+  const timelineM = new TimelineMax(box);
+  // TODO: why not chaining, why why why!
+  const tween1 = TweenMax.to(box, 1, { scale: 1.5, y: '+120', yoyo: true });
+  const tween2 = TweenMax.to(box, 1, { scale: 1, y: '-120', yoyo: true });
+  timelineM.repeat(-1);
+  timelineM.add([tween1, tween2], 0.5);
```

Comment:
@Caseyyu2 for chaining you should use the last parameter of `.add(*, *, *, stagger)` method. See: tutorial https://greensock.com/position-parameter and reference in API Docs https://greensock.com/docs/TimelineLite/add()

What we have in the current example - you are adding both animations and telling them to tween with half a second delay. They both are running simultaneously but the second one has overridable priority (last in the array and the latest in the call stack), so it's overriding `y` parameter from the second animation. Try to replace your animations with something like this to understand what I've meant:

```
const tween1 = TweenMax.to(box, 1, { y: '+120', yoyo: true });
const tween2 = TweenMax.to(box, 1, { scale: 1.5, yoyo: true });
```